### ISSUE TITLE
Handle Quarterly Data Properly

### ISSUE THREAD

**Issue by OliverSherouse** [OliverSherouse/bls#7]
Pandas chokes if the date string is, eg, "2017Q04". This makes sense, because there is no reason to zero pad a quarter, ever. Come on, BLS.

**Comment by jcress** [comment]
Like this, or something else?
```
    df.index = df.index.str.replace('-Q0', '-Q')
    df.index = pd.to_datetime(df.index)

```

**Comment by OliverSherouse** [comment]
Closed with #10
